### PR TITLE
LPS-176673 Replace alert divs with clay:alert taglib in site-admin-web module

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
The motivation behind this PR is to substitute the alerts created directly in html with clay:alerts taglib in the site-admin-web module.
https://issues.liferay.com/browse/LPS-176673

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/layout_set_merge_alert.jsp [hunk 1]
```diff
@@ -23,20 +23,20 @@ LayoutSetPrototype layoutSetPrototype = (LayoutSetPrototype)request.getAttribute
...
+			<liferay-ui:message arguments='<%= new Object[] {mergeFailCount, LanguageUtil.get(request, "site-template")} %>' key="the-propagation-of-changes-from-the-x-has-been-disabled-temporarily-after-x-errors" translateArguments="<%= false %>" />
 
-		<%
-		String randomNamespace = PortalUtil.generateRandomKey(request, "portlet_layout_set_prototypes_merge_alert") + StringPool.UNDERLINE;
-		%>
+			<liferay-ui:message arguments="site-template" key="click-reset-and-propagate-to-reset-the-failure-count-and-propagate-changes-from-the-x" />
 
-		<aui:button id='<%= randomNamespace + "resetButton" %>' useNamespace="<%= false %>" value="reset-and-propagate" />
-	</div>
+			<aui:button id='<%= randomNamespace + "resetButton" %>' useNamespace="<%= false %>" value="reset-and-propagate" />
```
Comment (sandrodw3) [inline modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/layout_set_merge_alert.jsp:37]: Could we take advantage and change this button to `clay:button`? 😄

**Review by sandrodw3 — COMMENTED** [review]
Hey @Stephy6! Looks good 😄I just suggested two improvements

**Review by sandrodw3 — COMMENTED** [review]
Hey @Stephy6! Looks good 😄I just suggested two improvements

**Comment by Stephy6** [comment]
ci:forward

**Review by sandrodw3 — COMMENTED** [review]
Hey @Stephy6! I left two new suggestions, please tell me what you think 🥳

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:stop

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**